app/migrations/helpers/protobuf_test.go

```diff
@@ -0,0 +1,19 @@
+package helpers
```

Comment:
Isn't it best practice according to ginkgo to add build comments so that tests aren't built? https://onsi.github.io/ginkgo/#recommended-continuous-integration-configuration

(This is a more general point I started looking at in my PR for workflows but it's worth discussing here.)